src/panels/config/users/dialog-user-detail.ts

```diff
@@ -55,31 +58,55 @@ class DialogUserDetail extends LitElement {
                 <div class="error">${this._error}</div>
               `
             : ""}
+          <div class="secondary">
+            ${this.hass.localize("ui.panel.config.users.editor.id")}: ${user.id}
+          </div>
+          <div>
+            ${user.is_owner
+              ? html`
+                  <span class="state"
+                    >${this.hass.localize(
+                      "ui.panel.config.users.editor.owner"
+                    )}</span
+                  >
+                `
+              : ""}
+            ${user.system_generated
+              ? html`
+                  <span class="state">
+                    ${this.hass.localize(
+                      "ui.panel.config.users.editor.system_generated"
+                    )}
+                  </span>
+                `
+              : ""}
+            ${user.is_active
+              ? html`
+                  <span class="state"
+                    >${this.hass.localize(
+                      "ui.panel.config.users.editor.active"
+                    )}</span
+                  >
+                `
+              : ""}
+          </div>
           <div class="form">
```

Comment:
I think we should add `padding-top: 16px` to this element. Separate it from the not editable fields above